http_post/sound_related/switch_party_mode.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+	<YAMAHA_AV cmd="PUT">
+		<System>
+			<Party_Mode>
+				<Mode>On</Mode>

```

Comment:
Thanks! But shouldn't it be

`<Mode>On | Off</Mode>`

or something like that?

Reply:
Ah, right, I have just taken the On example. Fixed now

Reply:
Perfect, thanks!